Xamarin.Forms.Platform.Android/Renderers/EditorRenderer.cs

```diff
@@ -11,7 +11,9 @@
 
 namespace Xamarin.Forms.Platform.Android
 {
-	public class EditorRenderer : ViewRenderer<Editor, FormsEditText>, ITextWatcher
+#pragma warning disable 618 // Have to continue using EditorEditText for now because third-party code may depend on it
```

Comment:
When I tried to fix that bug, I only added the old classes back in and didn't change the renderers. This also worked. So I don't know if you really need to change the renderers to use the old classes too.